app/assets/stylesheets/components/_organization_summary.scss

```diff
@@ -0,0 +1,42 @@
+.organization-summary {
+  $listing-height: 2 * $base-spacing;
+  background-color: $white;
+  padding: $base-spacing;
```

Comment:
Properties should be ordered background-color, box-shadow, margin-bottom, overflow, padding, position